woocommerce-services.php

```diff
@@ -209,6 +209,33 @@ function wpcom_static_url($file) {
 			return set_url_scheme( $url );
 		}
 
+		/**
+		 * environment_check function.
+		 *
+		 * @access public
+		 * @return void
+		 */
+		public function environment_check() {
+			if ( ! current_user_can( 'manage_woocommerce' ) ) {
+				return;
+			}
+
+			$admin_page = 'wc-settings';
+
+			$base_currency = get_woocommerce_currency();
+			$base_location = wc_get_base_location();
+
+			if ( ! WC_Connect_Shipping_Label::is_supported_currency( $base_currency ) ) {
+				echo '<div class="error">
+					<p>' . sprintf( __( 'WooCommerce Services requires that the <a href="%s">currency</a> is set to US Dollars.', 'woocommerce-services' ), admin_url( 'admin.php?page=' . $admin_page . '&tab=general' ) ) . '</p>
+				</div>';
+			} else if ( ! WC_Connect_Shipping_Label::is_supported_country( $base_location['country'] ) ) {
```

Comment:
using `else` here does not seem necessary since both currency and country could be unsupported so we should probably show both notices.
```suggestion
			} 
			if ( ! WC_Connect_Shipping_Label::is_supported_country( $base_location['country'] ) ) {
```